src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -835,16 +876,14 @@ void NonClientIslandWindow::_SetIsFullscreen(const bool fullscreenEnabled)
 
 // Method Description:
 // - Returns true if the titlebar is visible. For things like fullscreen mode,
-//   borderless mode, this will return false.
+//   focus mode, this will return false.

```

Comment:
if we're gonna keep the boolean/function names borderless in the Window classes, we should leave the comment saying "borderless" as well